ModPatches/Vanilla Arsenal/Defs/Vanilla Arsenal/Ammo.xml

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<parent>AmmoHighCaliber</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberHighCaliber</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<ammoTypes>
+			<Ammo_145x114mm_Demo>Bullet_145x114mm_Demo</Ammo_145x114mm_Demo>
+		</ammoTypes>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo145x114mmBase">
+		<defName>Ammo_145x114mm_Demo</defName>
+		<label>14.5x114mm bullet (Demo)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/HE</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<statBases>
+			<MarketValue>1.27</MarketValue>
+		</statBases>
+		<ammoClass>Demolition</ammoClass>

```

Comment:
This is a Thermobaric round, and should probably be labeled as such, using either `RocketThermobaric` or a new, non-rocket Thermobaric ammo class.